### PR TITLE
Docker healthcheck script

### PR DESCRIPTION
**Pull request by Starttoaster**

### CODE REVIEW

File: docker-healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    ;;
+    farmer-no-wallet)
+        node_check=true
+        farmer_check=true
+        harvester_check=true
+    ;;
+    farmer-only)
+        farmer_check=true
+    ;;
+    wallet)
```
Comment (cmmarslender) [inline docker-healthcheck.sh:42]: I think with 1.3.0, this only starts the wallet, and there is no wallet-only option

**Review by cmmarslender — CHANGES_REQUESTED** [review]

**Comment by Gnomuz** [comment]
Hi,
I reviewed the PR quickly as requested, and for my use case (remote harvester only), it should be just fine to query the harvester process with `get_plot_directories` to check the process health, as moreover it's not a heavy load request, even in the case of very large farms.
And I confirm the `wallet-only` option is deprecated with 1.3 and the embedded light wallet. `wallet` just starts the light wallet and syncs it with network peers (as the beta light wallet used to).

**Comment by cmmarslender** [comment]
> Hi, I reviewed the PR quickly as requested, and for my use case (remote harvester only), it should be just fine to query the harvester process with `get_plot_directories` to check the process health, as moreover it's not a heavy load request, even in the case of very large farms. And I confirm the `wallet-only` option is deprecated with 1.3 and the embedded light wallet. `wallet` just starts the light wallet and syncs it with network peers (as the beta light wallet used to).

Awesome. We're also looking into getting a health check endpoint added to the RPCs, so the endpoints in use might be even better in the future, we just tried to pick existing endpoints that should be pretty light calls for the time being.

File: docker-healthcheck.sh (hunk 1 above)
Comment (Starttoaster) [reply docker-healthcheck.sh:42]: Good callout. Fixing in a commit I have staged. Will push shortly.

Comment (Starttoaster) [reply docker-healthcheck.sh:42]: Fixed. Resolving thread

**Review by cmmarslender — APPROVED** [review]

**Pull request merged**